crypto/dsa/dsa_asn1.c

```diff
@@ -32,6 +32,15 @@ void DSA_SIG_get0(BIGNUM **pr, BIGNUM **ps, const DSA_SIG *sig)
     *ps = sig->s;
 }
 
+int DSA_SIG_set0(BIGNUM *r, BIGNUM *s, DSA_SIG *sig)
```

Comment:
In a FOO API, FOO should be the first argument: `DSA_SIG_set0(DSA_SIG *sig, BIGNUM *r, BIGNUM *s);` Ditto for ECDSA.

I have a vague recollection that @snhenson already added this API... but I'm probably misremembering?